### ISSUE TITLE
Rückgängigmachung von Aktenlöschung oder Dokumentenlöschung

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-client#182]
Habe gerade eine neu angelegte Akte versehentlich gelöscht. War nichts drin, Glück gehabt. Hilfreich wäre eine Rückgängigtaste zumindest der letzten 5 Aktionen...

**Comment by j-dimension** [comment]
Eine Rückfrage vor dem Löschen gibt es zumindest. In der Regel ist ein Löschen auch eher selten - es wäre somit z.B. auch denkbar den "normalen" Nutzern über die Nutzerverwaltung das Recht zum Akten löschen zu entziehen. Das dürte dann bspw. nur ein bestimmter Nutzer mit vollen Rechten. 

Eine Rückgängig-Taste müsste man dann schon konsistent überall umsetzen, das wäre signifikanter Aufwand.

**Comment by Mucki526** [comment]
Nein, braucht nicht bei jedem Schritt sein, aber eine gelöschte Akte (die letzten 3 oder 5) aus einer Historiie wiederherstellen? Oder eine Möglichkeit eine gelöschte Akte aus dem letzten Backup wiederherzustellen ohne das ganz große "Restore-Datenbank"-Faß aufzumachen....
